### PR TITLE
First pass of Helm Installation Documentation

### PR DESCRIPTION
**Pull request by George-Aeillo**

### CODE REVIEW

File: content/en/docs/setup/install/perform/helm-installation.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+1. Install your desired version of the Verrazzano VPO 
+   {{< clipboard >}}
+<div class="highlight">
+
+    $ helm install your-release-name oci://ghcr.io/verrazzano/charts/verrazzano-platform-operator --version 1.6.5
+
+</div>
+{{< /clipboard >}}
+
+This command installs only the 1.6.5 VPO to the cluster, it does not install a Verrazzano Custom Resource. To formally trigger an install of Verrazzano in your cluster, the custom resource [must be applied to your cluster]({{< relref "docs/setup/install/perform/kubectl-installation.md" >}}).
```
Comment (rosemarymarano) [inline content/en/docs/setup/install/perform/helm-installation.md:57]: -> Indent this paragraph 
-> This command installs only the VPO to the cluster; (semi-colon) it does not install a Verrazzano Custom Resource. 
-> 2. Follow the steps in this [section]({{< relref "docs/setup/install/perform/kubectl-installation#perform-the-installation" >}}) to apply the Verrazzano Custom Resource and complete the installation of Verrazzano.

**Review by rosemarymarano — COMMENTED** [review]
Some edits and suggestions.

File: content/en/docs/setup/install/perform/helm-installation.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+---
+title: "Install With Helm"
+description: "Install Verrazzano using Helm"
+weight: 3
+draft: false
+aliases:
```
[the remark this answers is not in the document]
Comment (George-Aeillo) [reply content/en/docs/setup/install/perform/helm-installation.md:6]: Thanks!

File: content/en/docs/setup/install/perform/helm-installation.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+- Find the Verrazzano prerequisite requirements [here]({{< relref "/docs/setup/install/prepare/prereqs.md" >}}).
+- Review the list of the [software versions supported]({{< relref "/docs/setup/install/prepare/prereqs.md#supported-software-versions" >}}) and [installed]({{< relref "/docs/setup/install/prepare/prereqs.md#installed-components" >}}) by Verrazzano.
+- [Helm](https://helm.sh)
+
+{{< alert title="NOTE" color="primary" >}}
+To avoid conflicts with Verrazzano system components, we recommend installing Verrazzano into an empty cluster.
+{{< /alert >}}
+
+{{< alert title="NOTE" color="primary" >}}
+Helm Chart Installation is only support for 1.6.0 releases and onward.
```
[the remark this answers is not in the document]
Comment (George-Aeillo) [reply content/en/docs/setup/install/perform/helm-installation.md:25]: Thanks!

File: content/en/docs/setup/install/perform/helm-installation.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+Make sure that you have a valid kubeconfig file pointing to the Kubernetes cluster that you want to use for installing your Verrazzano Platform Operator.
+
+**NOTE**: Verrazzano can create network policies that can be used to limit the ports and protocols that pods use for network communication. Network policies provide additional security but they are enforced only if you install a Kubernetes Container Network Interface (CNI) plug-in that enforces them, such as Calico. For instructions on how to install a CNI plug-in, see the documentation for your Kubernetes cluster.
+
+## Perform the installation
+
+Verrazzano provides a platform [operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/)
+to manage the life cycle of Verrazzano installations.
+
+This document shows how to install a VPO using:
```
[the remark this answers is not in the document]
Comment (George-Aeillo) [reply content/en/docs/setup/install/perform/helm-installation.md:40]: Thanks!

File: content/en/docs/setup/install/perform/helm-installation.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+* A Verrazzano VPO Helm chart, which can be found in one of our [repositories](https://github.com/orgs/verrazzano/packages/container/charts%2Fverrazzano-platform-operator/versions?filters%5Bversion_type%5D=tagged)
```
[the remark this answers is not in the document]
Comment (George-Aeillo) [reply content/en/docs/setup/install/perform/helm-installation.md:42]: Thanks!

File: content/en/docs/setup/install/perform/helm-installation.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+#### Install Verrazzano Platform Operator
```
[the remark this answers is not in the document]
Comment (George-Aeillo) [reply content/en/docs/setup/install/perform/helm-installation.md:44]: Thanks!

File: content/en/docs/setup/install/perform/helm-installation.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+{{< /alert >}}
+
+{{< alert title="NOTE" color="primary" >}}
+Helm Chart Installation is only support for 1.6.0 releases and onward.
+{{< /alert >}}
+
+## Prepare for the installation
+
+Before installing a Verrazzano platform operator, see instructions on preparing [Kubernetes platforms]({{< relref "/docs/setup/install/prepare/platforms/" >}}).
+Make sure that you have a valid kubeconfig file pointing to the Kubernetes cluster that you want to use for installing your Verrazzano Platform Operator.
```
[the remark this answers is not in the document]
Comment (George-Aeillo) [reply content/en/docs/setup/install/perform/helm-installation.md:31]: Thanks!

File: content/en/docs/setup/install/perform/helm-installation.md (hunk 1 above)
Comment (George-Aeillo) [reply content/en/docs/setup/install/perform/helm-installation.md:57]: Thanks!

File: content/en/docs/setup/install/perform/helm-installation.md [hunk 8]
```diff
@@ -37,13 +34,9 @@ Make sure that you have a valid kubeconfig file pointing to the Kubernetes clust
...
-
-* A Verrazzano VPO Helm chart, which can be found in one of our [repositories](https://github.com/orgs/verrazzano/packages/container/charts%2Fverrazzano-platform-operator/versions?filters%5Bversion_type%5D=tagged)
+The following steps show you how to install the VPO using a VPO Helm chart. VPO Helm charts are located in Verrazzano versioned repositories [here](https://github.com/orgs/verrazzano/packages/container/charts%2Fverrazzano-platform-operator/versions?filters%5Bversion_type%5D=tagged).
 
-#### Install Verrazzano Platform Operator
-
-To install a VPO Using a Helm Chart From Our Repository.
+#### Install the VPO using a Helm chart from the repository
 
 1. Install your desired version of the Verrazzano VPO
```
Comment (rosemarymarano) [inline content/en/docs/setup/install/perform/helm-installation.md:41]: -> 1. Install your desired version of the Verrazzano VPO. (period) See the following example command that installs the Verrazzano 1.6.5 VPO.

**Review by rosemarymarano — COMMENTED** [review]
Great job, George. You got in all my edits except one (maybe I didn't submit it), so I put it in this review.

**Review by bryantidd — APPROVED** [review]

**Review by rosemarymarano — APPROVED** [review]

**Pull request merged**